server/models/MenuItem.js

```diff
@@ -18,5 +18,39 @@ const getMenu = params => MenuItem.find(params)
   .then(data => (JSON.stringify(data)))
   .catch((err) => { throw err; });
 
+  const getMenuByName = params => MenuItem.find(params)
+  .then(data => (JSON.stringify(data)))
+  .catch((err) => { throw err; });
+
+  const getAllMenus = params => MenuItem.find()
+  .then(data => (JSON.stringify(data)))
+  .catch((err) => { throw err; });
+
+  const deleteMenu = params => MenuItem.deleteMany(params)
+  .then(data => (JSON.stringify(data)))
+  .catch((err) => { throw err; });
+
+  const createMenu = params => MenuItem.create(params)
+  .then(data => (JSON.stringify(data)))
+  .catch((err) => { throw err; });
+
+  const updateMenu = (params) => {
+    const modelId = params._id;
+    const newMenu = params.menu;
+    const newName = params.name;
+    const newDescription = params.description;
+    const newPrice = params.price;
+    const newCategory = params.category;
+
+    MenuItem.update({_id: modelId},{$set:{menu:newMenu,name:newName, description: newDescription, price: newPrice, category: newCategory }},{multi:true,new:true})
```

Comment:
You could just pass in the `params` object as the second argument to `update()`. No need to go through each property individually.

Reply:
TIL :)